WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController/NotificationsViewController.swift

```diff
@@ -540,15 +531,37 @@ private extension NotificationsViewController {
     }
 
     func updateNavigationItems() {
-        var barItems: [UIBarButtonItem] = []
+        self.navigationItem.rightBarButtonItem = moreMenuNavigationBarButtonItem
+        self.updateMoreMenuNavigationItem()
+    }
 
-        if shouldDisplaySettingsButton {
-            barItems.append(settingsBarButtonItem)
+    func updateMoreMenuNavigationItem() {
```

Comment:
Instead of updating the menu items whenever a change occurs, we can use `UIDeferredMenuElement.uncached` to defer the menu creation to when the button is tapped.

Ref: https://developer.apple.com/documentation/uikit/uideferredmenuelement

Reply:
Great suggestion @hassaanelgarem TIL

Reply:
Resolved in 2ea3c54cf26ad74429cb6c12c5c6fe08e5bae052